lib/presentation/enum/settings/settings_enum.dart

```diff
@@ -41,7 +41,7 @@ enum SettingEnum {
   String subtitleSettings(BuildContext context) {
     switch (this) {
       case SettingEnum.chatSettings:
-        return L10n.of(context)!.settingsChatSubtitle;
+        return '';
```

Comment:
why we don't delete SettingEnum.chatSettings if we don't use it anymore

Reply:
I don't think it's necessary, it's okay to update the value later